src/dcat-us/index.test.ts

```diff
@@ -53,4 +54,86 @@ describe('generating DCAT-US 1.1 feed', () => {
     expect(chk1.distribution).toBeInstanceOf(Array);
     expect(chk1.distribution.length).toBe(6);
   });
+
+  it('respects dcat customizations of overwritable attributes', async function () {
+    const feed = await generateDcatFeed(
+      mockSiteModel.item, 
+      [datasetFromApi],
+      {
+        description: '{{name}}', // overwrite existing attribute
+        customField: '{{name}}' // add new attribute
+      }
+    );
+
+    const chk1 = feed['dataset'][0];
+
+    expect(chk1['@type']).toBe('dcat:Dataset');
+    expect(chk1.identifier).toBe('https://download-test-qa-pre-a-hub.hubqa.arcgis.com/datasets/f4bcc1035b7d46cba95e977f4affb6be_0');
+    expect(chk1.license).toBe(null);
+    expect(chk1.landingPage).toBe('https://download-test-qa-pre-a-hub.hubqa.arcgis.com/datasets/f4bcc1035b7d46cba95e977f4affb6be_0');
+    expect(chk1.title).toBe('Tahoe places of interest');
+    expect(chk1.description).toBe('Tahoe places of interest');
+    expect(chk1.customField).toBe('Tahoe places of interest');
+    expect(chk1.keyword).toEqual([ 'Data collection', 'just modified' ]);
+    expect(chk1.issued).toBe('2021-01-29T15:34:38.000Z');
+    expect(chk1.modified).toBe('2021-07-27T20:25:19.723Z');
+    expect(chk1.publisher).toEqual({ name: 'QA Premium Alpha Hub' });
+    expect(chk1.contactPoint).toEqual({ '@type': 'vcard:Contact', fn: 'thervey_qa_pre_a_hub' });
+    expect(chk1.accessLevel).toBe('public');
+    expect(chk1.spatial).toBe('-121.118,38.7754,-119.009,39.359');
+    expect(chk1.theme).toEqual(['geospatial']);
+
+    expect(chk1.distribution).toBeInstanceOf(Array);
+    expect(chk1.distribution.length).toBe(6);
+  });
+
+  it('formats catalog correctly', async function () {
```

Comment:
Well, that's not supposed to be there 😅

Reply:
???

Reply:
I thought I had only copied one test, but it looks like the `it('formats catalog correctly')` test got duplicated on accident @andrewctate